src/pytorch_lightning/utilities/rank_zero.py

```diff
@@ -52,52 +44,37 @@ def _get_rank(trainer: Optional["pl.Trainer"] = None) -> Optional[int]:
 
 
 # add the attribute to the function but don't overwrite in case Trainer has already set it
-rank_zero_only.rank = getattr(rank_zero_only, "rank", _get_rank() or 0)
-
-
-def _info(*args: Any, stacklevel: int = 2, **kwargs: Any) -> None:
-    if python_version() >= "3.8.0":
-        kwargs["stacklevel"] = stacklevel
-    log.info(*args, **kwargs)
-
-
-def _debug(*args: Any, stacklevel: int = 2, **kwargs: Any) -> None:
-    if python_version() >= "3.8.0":
-        kwargs["stacklevel"] = stacklevel
-    log.debug(*args, **kwargs)
+new_rank_zero_only.rank = getattr(new_rank_zero_only, "rank", _get_rank() or 0)
 
 
-@rank_zero_only
+# TODO(lite): Does stacklevel need to be increased?
```

Comment:
Yes, by 1 (the new deprecation level added).

`tests/tests_pytorch/utilities/test_warnings.py` should be failing